www/docs/client/aborting-procedures.md

```diff
@@ -2,49 +2,9 @@
 id: aborting-procedure-calls
 title: Aborting Procedure Calls
 sidebar_label: Aborting Procedure Calls
-slug: /aborting-procedure-calls
+slug: /client/aborting-procedure-calls
```

Comment:
Seems a bit arbitrary to change this? I don't mind it, but then we should go through all "client"-stuff and rename them. Also we should add a redirect

tl;dr - I think the url change shouldn't be part of this PR and instead done on later PR that changes _all_ routes

Reply:
Just to pop deets here, yes let's separate out the router changes for existing pages into an initial PR, I will do it once I'm happy with this initial re-org and know the full scope :)

For the "why" - we have URL conflicts now, because categorisation and consistency are important. This page doesn't only document the vanilla client so I've moved the react stuff to the react category